--- makemake.orig	2014-11-25 14:26:28.468283898 +0100
+++ makemake	2014-11-25 14:26:49.832283851 +0100
@@ -622,7 +622,7 @@
 # Fortran compiler and compilation flags
 #
 FCOMPL=$FCOMPL
-FFLAGC=$FFLAGC
+FFLAGC=$FFLAGC -fPIC
 FFLAGD=$FFLAGD
 #
 # C compiler and compilation flags
@@ -633,7 +633,7 @@
 TK_INCL=$TK_INCL
 RV_INCL=$RV_INCL
 CCOMPL=$CCOMPL
-CFLAGC=$CFLAGC -I.
+CFLAGC=$CFLAGC -I. -fPIC
 CFLAGD=$CFLAGD
 MCOMPL=$MCOMPL
 MFLAGC=$MFLAGC
@@ -1042,12 +1042,13 @@
 pgbind: $(SRC)/cpg/pgbind.c
 	$(CCOMPL) $(CFLAGC) $(SRC)/cpg/pgbind.c -o pgbind
 
-libcpgplot.a cpgplot.h: $(PG_SOURCE) pgbind 
+libcpgplot.a libcpgplot.so cpgplot.h: $(PG_SOURCE) pgbind 
 	./pgbind $(PGBIND_FLAGS) -h -w $(PG_SOURCE)
 	$(CCOMPL) -c $(CFLAGC) cpg*.c
 	rm -f cpg*.c
 	ar ru libcpgplot.a cpg*.o
 	$(RANLIB) libcpgplot.a
+	$(CCOMPL) -shared -o libcpgplot.so cpg*.o
 	rm -f cpg*.o
 
 cpgdemo: cpgplot.h $(SRC)/cpg/cpgdemo.c libcpgplot.a
